portfolio/src/main/webapp/script-home.js

```diff
@@ -49,3 +49,12 @@ function main() {
           slideShowGallery.adjustSlideManual(ADJUST_FORWARD); 
     }
 }
+
+// A function for a button click, changes the button to header, Who Am I, 
```

Comment:
nit: use a consistent comment style (above methods are using /* */)

It might be worth a JSDoc comment here as well.

Reply:
I think function comments are /**/ but inline code comments are "//" which I think is pretty standard?